### ISSUE TITLE
[Warning] The purpose of each link should be clear from the link text (18)

### ISSUE THREAD

**Issue by rodowi** [mxabierto/participa#231]
https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_04
```html
<a href="" ng-click="selectPage(page.number)" class="ng-binding">1</a>
```

**Comment by rodowi** [comment]
Encontrado en:
- /consulta
-  /consulta/auth/login

**Comment by codemoe** [comment]
@rodowi Éste issue hace referencia a los links del paginator... Actualmente hay un issue abierto en GoogleChrome/accessibility-developer-tools sobre éste tema. https://github.com/GoogleChrome/accessibility-developer-tools/issues/240

Creo que lo dejaremos pendiente ya que no hay mucho por hacer con éste tema

**Comment by rodowi** [comment]
Lo mandaré al back-backlog entonces
